pandas/tslib.pyx

```diff
@@ -2416,6 +2416,8 @@ cdef inline parse_timedelta_string(object ts, coerce=False):
     elif have_dot:
 
         if (len(number) or len(frac)) and not len(unit) and current_unit is None:
+            if coerce:
+                return iNaT
```

Comment:
hmm, what exactly is testing this?

as this is not allowed (and ``Timedelta`` doesn't have a coerce option itself)
```
In [1]: pd.Timedelta('1.')
ValueError: no units specified
```